src/github/push.unit.test.ts

```diff
@@ -0,0 +1,69 @@
+import { pushWebhookHandler } from "./push";
```

Comment:
Is the name `*.unit.test.ts` a new convention? I'm sure many of our existing `*.test.ts` are unit tests, too.

Reply:
This is a bit funny, as the current `push.test.ts` under the same folder is actually testing things somewhere else, it is NOT testing anything `push.ts` in the same directory. So I've to create another one to unit test it. Leave the current one untouched.

Reply:
Which test are you talking about? There doesn't appear to be any other test for push in that directory 😕 

![Screen Shot 2022-08-10 at 1 06 48 pm](https://user-images.githubusercontent.com/37155488/183801530-a434dac3-5268-4877-ac42-c158efa84f30.png)

Reply:
Remove origin file to sqs/push.test.ts

Reply:
fixed.